core/src/main/java/com/datastax/oss/driver/internal/core/os/Native.java

```diff
@@ -34,172 +23,53 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(Native.class);
 
+  private static class ImplLoader {
+
+    public Libc load() {
+      try {
+        return new JnrLibc();
+      } catch (Throwable t) {
+        LOG.info(
+            "Unable to load JNR native implementation. This could be normal if JNR is excluded from the classpath",
+            t);
+        return new EmptyLibc();
+      }
+    }
+  }
+
+  private static final Libc IMPL = new ImplLoader().load();
```

Comment:
We can rename to `LIBC` and `LibcLoader` now that we have a better name for the interface.

Reply:
Agreed